README.md

```diff
@@ -23,6 +26,37 @@ This repository contains an SDK and Jupyter notebooks with examples to showcase
 - [Build Status](#BuildStatus)
 - [Additional References](#Refs)
 
+# <a name="intro"></a> 
+
+# Overview of Interpret-Community

```

Comment:
in the contents, it's interpret-community and here it's Interpret-Community
good to pick one or the other